Core1Platform/UserInterface/02_Dashboard.md

```diff
@@ -0,0 +1,211 @@
+
+In the *Actindo Dashboard*, the custom tabs with composable dashlets are displayed. By default no tab is created.
+
+- ![Add](../../Assets/Icons/Plus06.png "[Add]") (Add)   
+    Click this button to add a new tab to display different dashlets. The *Add a new tab* window is displayed.
+
+- ![Delete](../../Assets/Icons/Cross04.png "[Delete]") (Delete)   
+    Click this button to delete the selected tab. The *Confirm tab deletion* window is displayed. This button is displayed to the right of the tab name.
+
+
+
+## Add tab
+
+*Actindo Dashboard > Button Add*
+
+![Create new tab](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardAddNewTab.png "[Create new tab]")
+
+**Create tab**
+
+- *Tab Name*   
+    Enter a name for the new tab.
+
+- [CANCEL]   
+    Click this button to cancel creating a tab. The *Add a new tab* window is closed.
+
+- [SAVE]   
+    Click this button to save the tab with the entered name. The *Add a new tab* window is closed. The new tab is created.
+
+**Import tab**
+
+![Import tab](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardImport.png "[Import tab]")
+
+- *Dashboard selection*   
+    Select a module-specific dashboard that you have already created before.   
+
+
+- [CANCEL]   
+    Click this button to cancel creating a tab. The *Add a new tab* window is closed.
+
+- [SAVE]   
+    Click this button to save the tab with the entered name. The *Add a new tab* window is closed. The new tab is created.
+
+
+
+## Confirm tab deletion
+
+*Actindo Dashboard > Button Delete*
+
+![Confirm tab deletion](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardConfirmTabDelition.png "[Confirm tab deletion]")
+
+- [CANCEL]   
+    Click this button to cancel deleting the selected tab. The *Confirm tab deletion* window is closed.
+
+- [ACCEPT]   
+    Click this button to delete the selected tab. The *Confirm tab deletion* window is closed. The tab and its dashlets are deleted.
+
+
+
+## Tab
+
+*Actindo Dashboard > Select tab*   
+*Actindo Dashboard > Button Add > Button Save*   
+
+![New tab](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardNewTab.png "[New tab]")
+
+ ![Settings](../../Assets/Icons/Settings01.png "[Tool]") (Settings)   
+Click this button to edit the tab. The *Edit tab* view is displayed.  
+
+
+### Edit tab
+
+*Actindo Dashboard > Select tab > Button Settings*   
+*Actindo Dashboard > Button Add > Button Save > Button Settings*  
+
+![Edit tab](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardEditTab.png "[Edit tab]")
+
+- *Select Viewport*   
+    Click the drop-down list to select the view to be applied to the tab. A different view can be selected for each tab. The drop-down list is only displayed in the *Edit tab* view. The following options are available:    
+    - **Automatic**   
+        The system automatically chooses the dashboard that has a suitable size for the current device.
```

Comment:
Ist das der "Dashboard" oder die Ansicht des Dashbaords, den/die vom System abhängig von Gerät auswählt wird. Für die unteren Beschreibungen: ist das "Größe des Dashboards" oder "Layout/Ansicht"? Im Sandbox sehe ich kaum unterschied... In DataHub Dashboard wird anders erklärt. Evtl. Julian/Entwicklung fragen, wie es gemeint ist?

Reply:
hatte Julian schon gefragt